integration_tests/mockData/prisonerByIdData.ts

```diff
@@ -8,7 +8,7 @@ const prisoners = {
       status: 200,
       headers: { 'Content-Type': 'application/json;charset=UTF-8' },
       jsonBody: {
-        prisonId: 'MDI',
+        prisonId: 'BXI',
```

Comment:
These prisoner's in the wiremock stub data (for the cypress integration tests) have all been changed to BXI (Brixton) because the user that the cypress tests use has an active caseload ID of BXI